### DISCUSSION TITLE
How to set sign_in_after_reset_password to false?

### DISCUSSION THREAD

**Discussion by kirstenland** [devise-two-factor/devise-two-factor#217] [Q&A]
Not an issue, just a question

In the README, in the section about password resetting, it says: "Because of this, you need to set sign_in_after_reset_password to false (either globally in your Devise initializer or via devise_for)". I don't want to do this globally as some of my user-types don't have 2fa. 

I've tried
```
  devise_for :admin_users, ActiveAdmin::Devise.config.merge(sign_in_after_reset_password: false)
```

but it doesn't seem to be working and I still get signed in after a password reset.

(I'm also using active admin, I don't know if this is related)

**Comment by eimantas** [comment]
You should set this in devise initializer at `config/initializers/devise.rb`:

```ruby
  config.sign_in_after_reset_password = false
```
